src/doblib/migrate.py

```diff
@@ -0,0 +1,71 @@
+# © 2021-2022 Ruben Ortlam (initOS GmbH)
+# License Apache-2.0 (http://www.apache.org/licenses/).
+
+from contextlib import closing
+
+from . import base, utils
+from .aggregate import AggregateEnvironment
+from .module import ModuleEnvironment
+from .run import RunEnvironment
+
+
+def load_migrate_arguments(args):
+    parser = utils.default_parser("init")
+    parser.add_argument(
+        "version",
+        default=[],
+        help="Target Odoo version",
+    )
```

Comment:
`type=utils.Version`. `required`?

Please add an example in the help (e.g. if you expect 15 or 15.0)